index.html

```diff
@@ -712,101 +712,79 @@ <h2 class='informative'>Vary</h2>
+          Implementers should be aware that some operations cause multiple resources to change.
+          In these cases, there will be a corresponding notification describing each of the changes.
+          This is especially true for changes to containment or membership triples. This is also true
+          if a <code>DELETE</code> operation triggers changes in any contained resources.
         </p>
         <p>
-          Any operation on an <a>LDPR</a> that results in a change to the containment or membership
-          triples of a distinct other <a>LDPR</a> MUST also trigger an event for that other <a>LDPR</a>.
+          Consumers of these notifications should not expect a strict ordering of
+          the events reported therein: the fact that a notification for Event A is received before
+          a notification for Event B should not imply that Event A occurred before Event B.
+          Implementations may choose to make further guarantees about ordering.
         </p>
         <p>
-          Any operation that causes contained resources to change MUST trigger corresponding
-          events for those resources. For example, deleting a parent resource and thereby
-          changing the state of its child resources MUST result in corresponding events for
-          each of the contained child resources.
-        </p>
-        <p class="informative">
-          Non-normative: consumers of these messages should not expect a strict ordering of
-          the events reported therein: the fact that a message for Event A is received before
-          a message for Event B should not imply that Event A occurred before Event B.
-          Implementations may choose to make further guarantees about ordering.
+          According to the [[activitystreams-core]] specification, it is possible to collect multiple
+          events into a single notification. This specification makes no restriction on the use of
+          activity stream collections.
         </p>
       </section>
 
-      <section id="message-data">
-        <h3>Message Data</h3>
-        <p>
-          Emitted events MUST contain exactly one value for each of the following elements:
-        </p>
-        <ul>
-          <li>Resource Identifier: the IRI of the <a>LDPR</a> that was added, changed or removed.</li>
-          <li>Event identifier: a unique identifier for this event.</li>
-        </ul>
+      <section id="notification-events">
+        <h3>Notification Events</h3>
         <p>
-          A message MUST contain at least one value for the event type. Event types SHOULD be drawn
-          from the [[!activitystreams-vocabulary]].
-        </p>
-        <p>
-          A message SHOULD contain at least one value for the agent operating on the resource.
-        </p>
-        <p>
-          A message SHOULD contain a collection of RDF types corresponding to the resource that
```

Comment:
The current language is:

> A message must contain at least one value for the event type. Event types should be drawn from the [activitystreams-vocabulary].

The new language is:

>  Wherever possible, data SHOULD be expressed using the [[!activitystreams-vocabulary]].

and

> Each event described by a notification MUST contain ... the event type(s), corresponding to the HTTP operation

How is that dropping the `MUST` language?